mmedit/models/editors/stylegan2/stylegan2_generator.py

```diff
@@ -262,6 +305,8 @@ def get_mean_latent(self, num_samples=4096, **kwargs):
         Returns:
             Tensor: Mean latent of this generator.
         """
+        if hasattr(self, 'w_avg'):
```

Comment:
Can use decide which mean latent they get? Or may be we can tell them the way we get mean latent by hints.

Reply:
An argument named `update_mean_latent_with_ema` is added to initialize function of StyleGAN2's generator.

Reply:
may be we can tell them the way we get mean latent by hints.